packages/client/index.js

```diff
@@ -189,6 +195,29 @@ function buildCallFunction (baseUrl, path, method, methodMeta, throwOnError, ope
           body: responseBody
         }
       }
+
+      if (validateResponse) {
+        try {
+          // validate response first
+          const matchingResponse = responses[res.statusCode]
+          
+          if (matchingResponse === undefined) {
+            throw new Error(`No matching response schema found for status code ${res.statusCode}`)
+          }
+          const matchingContentSchema = matchingResponse.content[contentType]
+          
+          if (matchingContentSchema === undefined) {
+            throw new Error(`No matching content type schema found for ${contentType}`)
+          }
+          const bodyIsValid = checkResponseAgainstSchema(responseBody, matchingContentSchema.schema)
```

Comment:
Does the schema have all the refs already resolved? Could you add a test with some shared schemas?